docker/rococo-parachain-launch-config.yml

```diff
@@ -13,13 +13,13 @@ relaychain:
     - --execution=wasm
   nodes:
     - name: alice
-      wsPort: 9946
-      rpcPort: 9936
-      port: 30336
+      wsPort: ${AliceWSPort:-9946}
+      rpcPort: ${AliceRPCPort:-9936}
+      port: ${AlicePort:-30336}
```

Comment:
Adding the defaults here sounds a bit dangerous imo; someone can forget to set up their local env files and the defaults here will fall back to the Forbidden Ports instead of rightfully complaining :D

Reply:
This for the CI basically, if we don't add it here then we need to maintain separate config files for development and CI